extensions/map-layers-auth/src/ArcGis/ArcGisOauthRedirect.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { ProgressRadial } from "@itwin/itwinui-react";
```

Comment:
Not seeing `itwinui-react` as a dependency for the auth package

Reply:
its part of `devDependencies`.  I guess I should add it to `peerDependencies`, right?

Reply:
you should be able to add it as a real dependency based on the way you're using it.